contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3502,8 +3502,7 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 
 				if (drop_stmt->removeType != OBJECT_SCHEMA)
 				{
-					if (sql_dialect == SQL_DIALECT_TSQL)
```

Comment:
Why is this change needed?

Reply:
This change is a part of discussion here - https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/2708#discussion_r1677347072